app/imageswap-init/imageswap-init.py

```diff
@@ -53,13 +53,12 @@
 imageswap_tls_byoc = False
 imageswap_pks_namespace = "pks-system"
 
-###############################################################################$
+# $
```

Comment:
Not sure what happened here, but this line should match the lines beneath for the existing formatting